src/components/common/EmulatorConfigProvider.tsx

```diff
@@ -234,6 +234,10 @@ async function configFetcher(url: string): Promise<Config> {
           // resolve it to IPv6 and connection may fail.
           host = '127.0.0.1';
         }
+        if (key == 'firestore') {
+          // If we remapped the host, apply it to the websocket host too.
+          value.webSocketHost = value.host;
```

Comment:
should this be host instead of `value.host`?

Reply:
Good catch, done

Reply:
I don't see the update here. Did you forget to push?